src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksWebServiceCallsServiceImpl.java

```diff
@@ -48,9 +44,8 @@
 import edu.cornell.kfs.sys.CUKFSConstants;
 import edu.cornell.kfs.sys.service.WebServiceCredentialService;
 import edu.cornell.kfs.sys.util.CURestClientUtils;
-import edu.cornell.kfs.sys.web.CuMultiPartWriter;
 
-public class PaymentWorksWebServiceCallsServiceImpl implements PaymentWorksWebServiceCallsService, Serializable {
+public class PaymentWorksWebServiceCallsServiceImpl extends DisposableClientServiceImplBase implements PaymentWorksWebServiceCallsService, Serializable {
```

Comment:
Now that this service relies on the lazy-loaded Client feature, please update the corresponding PaymentWorks unit tests so that they call the destroy() method afterwards to clean up the Client instance.